pkg/repo/fs.go

```diff
@@ -94,14 +90,6 @@ func (fsr *FsRepo) Init(cfg types.Bacalhau) error {
 
 	log.Info().Msgf("Initializing repo at %s", fsr.path)
 
-	// if it takes longer than 5 seconds to get the node name from a provider, fail
-	nameCtx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
-	defer cancel()
-	nodeName, err := getNodeID(nameCtx, cfg.NameProvider)
-	if err != nil {
-		return err
-	}
-
```

Comment:
since a node name is only needed by the `serve` command I have removed this code, it's now only done in `cmd/cli/serve.go`